libbmp.c

```diff
@@ -207,7 +207,7 @@ bmp_img_read (bmp_img    *img,
 	for (size_t y = 0; y < h; y++)
 	{
 		// Read a whole row of pixels from the file:
-		if (fread (img->img_pixels[abs (offset - y)], sizeof (bmp_pixel), items, img_file) != items)
+		if (fread (img->img_pixels[offset - y], sizeof (bmp_pixel), items, img_file) != items)
```

Comment:
This isn't functionally correct. If `offset = 0` then the value wraps around starting from the max value of `size_t`